molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/algorithmgenerator/generator/OneToManyCategoryAlgorithmGenerator.java

```diff
@@ -16,10 +16,11 @@
 import org.apache.commons.lang3.StringUtils;
 import org.molgenis.data.AttributeMetaData;
 import org.molgenis.data.DataService;
+import org.molgenis.data.EntityMetaData;
 import org.molgenis.data.mapper.algorithmgenerator.bean.AmountWrapper;
 import org.molgenis.data.mapper.algorithmgenerator.bean.Category;
 
-public class OneToManyCategoryAlgorithmGenerator extends CategoryAlgorithmGenerator
+public class OneToManyCategoryAlgorithmGenerator extends AbstractCategoryAlgorithmGenerator
 {
 	private static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("##.#",
```

Comment:
The constructor for the Locale class should be like this **new Locale(language)** , this line of code is correct, however in the selenuim test it was wrong indeed. Fleur has largely rewritten the seleinum tests and this piece of code has gone in the master.